ui/app/components/mount-backend-form.ts

```diff
@@ -27,7 +28,7 @@ import type { MountSecretBackendModel } from 'vault/routes/vault/cluster/setting
  *   <MountBackendForm @mountType="secret" @onMountSuccess={{this.onMountSuccess}} />```
  *
  * @param {function} onMountSuccess - A function that transitions once the Mount has been successfully posted.
- * @param {string} [mountType=auth] - The type of backend we want to mount.
+ * @param {string} mountType=auth - The type of engine to mount, either 'secret' or 'auth'.
```

Comment:
Commented in a test already, but could this be updated too so that we're consistent across the board?
```suggestion
 * @param {string} mountCategory=auth - The type of engine to mount, either 'secret' or 'auth'.
```